lib/devices/Roborock.js

```diff
@@ -16,6 +16,8 @@ class Roborock extends MiioVacuum {
          * @type {{[id: string]: {label: string, value: number}}}
          */
         this.fanSpeeds = null;
+        /** @private */
```

Comment:
Add some comments on what this means / is being used for.
And potentially when using the setting latter, add some comments about the differences in command set (e.g. pre-gen3 don’t take arguments for set_timer?).

Reply:
done